tests/passing/brils/fib-2-unroll.bril

```diff
@@ -0,0 +1,46 @@
+# ARGS: 5
```

Comment:
the brils folder is for tests from the bril repo- you can put these in "other"
or "small" if you want snapshot testing